config/install/administerusersbyrole.settings.yml

```diff
@@ -2,4 +2,5 @@ roles:
   school: unsafe
   school_admin: unsafe
```

Comment:
We should also update school, school_admin. Since we are updating these roles in the install file.